### PR TITLE
Add FooterName and HeaderName to mirror httpd

### PR DESCRIPTION
**Pull request by olblak**
We'll generate htlm files that can be display before and after directory listing as decscribe [here](https://github.com/jenkinsci/packaging/pull/159)

Following configuration ne3ed to be updated
- [x] mirror.azure.jenkins.io
- [x] archives.azure.jenkins.io
- [x] get.jenkins.io

### CODE REVIEW

**Comment by olblak** [comment]
The mirrorbits chart need to be refactored in order to split the file section into a different chart but for now...

File: charts/mirror/templates/configmap.yaml [hunk 1]
```diff
@@ -248,10 +248,14 @@ data:
         #
         Require all granted
 
-        #IndexHeadInsert "<link rel=\"stylesheet\" href=\"https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css\" integrity=\"sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh\" crossorigin=\"anonymous\">" 
-        IndexHeadInsert "<link rel=\"stylesheet\" href=\"https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css\">"
-
         IndexOptions FancyIndexing HTMLTable VersionSort TrackModified FoldersFirst SuppressDescription
+
+        IndexOptions FancyIndexing VersionSort TrackModified FoldersFirst SuppressHTMLPreamble NameWidth=* SuppressRules SuppressIcon SuppressDescription
```
Comment (timja) [inline charts/mirror/templates/configmap.yaml:253]: do you need the IndexOptions on the line above still?

File: charts/env-jenkins-release/templates/core-packages.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ spec:
   storageClassName: azurefile
```
Comment (timja) [inline charts/env-jenkins-release/templates/core-packages.yaml:22]: swap file again

File: charts/mirror/templates/configmap.yaml (hunk 1 above)
Comment (olblak) [reply charts/mirror/templates/configmap.yaml:251]: thanks for catching this duplicated line

**Pull request merged**